sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -124,6 +126,29 @@ abstract class JdbcDialect extends Serializable {
   }
 
   /**
+   * Make string literal in SQL valid.
```

Comment:
`Make string literal in SQL valid.`
->
`Escape special characters in SQL string literals.`